test/functional/feature_anchors.py

```diff
@@ -7,11 +7,14 @@
 import os
 
 from test_framework.p2p import P2PInterface
```

Comment:
why was P2P_SERVICES not added here?

Reply:
Please refer to pull request description for reasoning.